integrations/tensorflow/e2e/keras/iree_vision_test_suite.bzl

```diff
@@ -18,61 +18,78 @@ load("//bindings/python:build_defs.oss.bzl", "iree_py_test")
 
 def iree_vision_test_suite(
         name,
-        configurations,
+        reference_backend,
+        datasets,
+        models_to_backends,
         external_weights = None,
-        deps = None,
         tags = None,
+        deps = None,
         size = "large",
         python_version = "PY3",
         **kwargs):
-    """Creates one iree_py_test per configuration tuple and a test suite that bundles them.
+    """Expands a set of iree_py_tests from these arguments into a test suite.
 
     Args:
-      name: name of the generated test suite.
-      configurations: a list of tuples of (dataset, include_top, model,
-                      backends) that specifies which data, model and backends to
-                      use for a given test.
-      external_weights: a base url to fetch trained model weights from.
-      tags: tags to apply to the test. Note that as in standard test suites,
-            manual is treated specially and will also apply to the test suite
-            itself.
-      size: size of the tests.
-      python_version: the python version to run the tests with. Uses python3
-                      by default.
-      **kwargs: Any additional arguments that will be passed to the underlying
-                tests and test_suite.
+      name:
+        name of the generated test suite.
+      reference_backend:
+        the backend to use as a source of truth for the expected output results.
+      datasets:
+        a list specifying which model weights to load for external tests. Also
+        specifies the shape of the random images to use for testing.
```

Comment:
Let's emphasize the input format first and then discuss the special case of external weights.

```suggestion
      datasets:
        a list specifying the dataset on which the model is based. This controls
        the shape of the input images. Also indicates which weight file to use
        when loading weights from an external source.
```

(and then put in the appropriate line breaks)